webapp/src/js/filters/date.js

```diff
@@ -152,7 +156,8 @@ var _ = require('underscore'),
         RelativeDate: RelativeDate,
         withoutTime: true,
         age: true,
-        end: dod
+        end: dod,
+        class: 'age'
```

Comment:
The `class` property is now unused so it should be removed.

Reply:
I was sure I removed it but thanks for spotting it.